states/team.py

```diff
@@ -1,7 +1,33 @@
+from typing import Optional
 
+from config.config import TEXT
+from states.battle import Battle
+from states.state import State, StateOptions
 
-class Team:
+
+class TeamOptions(StateOptions):
+    """Enum values for options availabe in team menu"""
+
+    TOSSPKMN = 1
+    REORDER = 2
+    EXIT = 3
+
+
+class Team(State):
     """Game state for player team management"""
 
+    option: Optional[TeamOptions] = None
+
     def __init__(self):
         super().__init__()
+        while self.option is not TeamOptions.EXIT:
+            print(TEXT["TEAM"]["ENTRY"])
+            self.option = self.check_input(TeamOptions)
+            self.choose_option()
+
+    def choose_option(self):
+        """Enters next game state based on player choice"""
```

Comment:
When we choose an organization option, we're actually not going to create an entirely new game state just for the next step. We'll probably just enter a new function under the `class Team(State):`, and maybe offer a substate if there are more decisions to be made. At any rate whatever it is, just fix the docstring to describe that `choose_option` will perform the team action of their choice.

Reply:
Changed to `Performs team action based on player choice`